libs/ui/src/screens/wallet/components/assets-widget/assets-widget.styles.ts

```diff
@@ -4,7 +4,7 @@ import { getCustomSize } from '../../../../styles/format-size';
 
```

Comment:
we have an agreement to name branches with the `kl-numberOfTask`, for example
`kl-471/fix-margin-widgets`

and it will be automatically linked to jira ticket

Reply:
I don't know that, sorry(

BTW the branch, PR, and commit linked to the task automatically)